src/owo.js

```diff
@@ -1,34 +1,37 @@
-
 const { createReadStream } = require('fs');
 const Endpoints = require('./Endpoints');
 const request = require('superagent');
 const OwOError = require('./owoError');
 
-/*
- * Creates a new class
- * @prop {key} key your whats-th.is API Key.
- */
+/**
+  * OwO Client
```

Comment:
But this is JSDoc for the class, you document what it creates on constructor below.